git_pile/helpers.py

```diff
@@ -82,20 +82,21 @@ def __call__(self, s, *args, **kwargs):
         kwargs["check"] = kwargs.get("check", self.check)
         kwargs["shell"] = kwargs.get("shell", self.shell)
 
+        cmd = [self.cmd] if isinstance(self.cmd, str) else self.cmd.copy()
```

Comment:
yeah... we may want to skip the split if shell=true. The users are:

pre_genbranch_exec = run_wrapper(args.pre_genbranch_exec, shell=True)   
post_genbranch_exec = run_wrapper(args.post_genbranch_exec, shell=True) 

The only place I know using this is just passing a .sh as argument.... and it doesn't really need shell=True. I wonder if we should remove shell==True support or fix it.

Reply:
I think we could do things right when using `shell=True`, so we can support keywords supported by`subprocess.run`... Otherwise we will need to maintain a documented list of unsupported keywords.

Reply:
Ok, fixed in the next version